### ISSUE TITLE
Arm64 Support

### ISSUE THREAD

**Issue by Aeyk** [workadventure/workadventure#3800]
It seems like it's all cross platform capable languages, is there arm64 support?

**Comment by moufmouf** [comment]
Yes, we provide docker images and those images are built both for AMD64 and ARM64.

**Comment by Aeyk** [comment]
How can I run it? When I try to use the docker compose on linux/arm64, some
containers (play, back, iconserver, map-storage, uploader)  get exec error
like it's a wrong architectures binary.
